outscale/data_source_outscale_account.go

```diff
@@ -0,0 +1,155 @@
+package outscale
+
+import (
+	"context"
+	"time"
+
+	oscgo "github.com/outscale/osc-sdk-go/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-outscale/utils"
+)
+
+func dataSourceOutscaleOAPIAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceOutscaleOAPIAccountRead,
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"additional_emails": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"city": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"company_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"country": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"customer_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"first_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"job_title": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"last_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"mobile_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"phone_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"state_province": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"vat_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"zip_code": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"request_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceOutscaleOAPIAccountRead(d *schema.ResourceData, meta interface{}) error {
```

Comment:
`s/dataSourceOutscaleOAPIAccountRead/dataSourceAcountRead/`
We did use OutscaleOAPI in some function, but this is deprecated
The OAPI tell nothing about the function utility and repeat what `Outscale` already tell
Being insid Outscale Provider, rewrite Outscale in the Function name is also irrelevant. (but more up to debate)

Reply:
I'm ok with that, but actually every function is named like this, for me it's another issues for fix this in every file

Reply:
That's partially true, some function have remove the "OAPI". but most have both.
But yes renaming everything is still something we have to do, but a low priority issue.

Reply:
Done